additional info

riaars opened the pull request:
adjust responsive layout

Envl committed:
relocate storageHelper.js

Envl committed:
relocate storageHelper.js

Envl committed:
despose Toggle.js and modify Button.js

Envl committed:
toggle style

Envl committed:
change button-bg transition to darken color and remove focus animation

Envl committed:
Create @mixin smaller-than

Envl committed:
Responsiveness of BooksPage

Envl committed:
Delete redundant code. Minor improvement for toggles

Envl committed:
align all tool-bar buttons

Envl committed:
Create BookItem_v2
Refactored some small components from BooksPage to Books/Components.js

Envl committed:
Improve layout for mobile

Envl committed:
Fix bookitem click

Envl committed:
Make it hover to show menu on account button

Commit:
additional info

Envl committed:
Create History Page

bt-s reviewed (changes requested):
Good work!

One thing should be changed though:
```html, body { max-width: 100%; overflow-x: hidden;}``` is not unique to _BookDetails.scss.

This should be moved to src/styling/includes/_base.scss

Envl committed:
Styled Radio Buton

Envl committed:
Create and style Message in History Page

Envl committed:
Layout and style HistoryPage

Envl committed:
Make the data flow right and minor style fix
Note: the background of `div`s are deliberately made for debug purpose.

Envl committed:
Make local query working
Also deleted unnecessary state `showingMsg`

Envl committed:
delete log and fix typo

Envl committed:
refactor over nested _BookItem.scss file

Envl committed:
Merge pull request #46 from bt-s/style-refinement

Style refinement for Books Page

Envl committed:
Merge branch 'history-page'

Envl committed:
Merge pull request #47 from bt-s/history-page

History page

Envl committed:
Merge branch 'master' of https://github.com/bt-s/ipdw-project

bt-s committed:
Remove console log statement

bt-s committed:
Resolve browser warnings

bt-s committed:
Make navbar account icon clickable again

bt-s committed:
Move validation component to different folder (/Forms)

bt-s committed:
Fix validation issue in SignUp component

I have added a check to the onSubmit function, such that the firebase
signup promise only gets executed when all required values have been
filled out.

bt-s committed:
Change name of PasswordChange button

bt-s committed:
Fix linting errors

fadhilmch committed:
add full functionality on add_book_page

fadhilmch committed:
clean state on redux

fadhilmch committed:
add promise to storage helper

fadhilmch committed:
change title of add_book_page

bt-s committed:
Merge pull request #48 from bt-s/refactor

Refactor

bt-s committed:
Fix linting errors

Envl committed:
Merge branch 'master' of https://github.com/bt-s/ipdw-project

Envl committed:
Fix Navbar menu alwayshow issue

Envl committed:
Fix BookItem layout issue

Envl committed:
Change tag of file-drop-box to div to make it clickable

Envl committed:
resolve conflict

Envl committed:
git push origin masterMerge branch 'add_new_book_image_upload'

Envl committed:
resolve path error

Envl committed:
Merge pull request #49 from bt-s/add_new_book_image_upload

Add new book image upload

Envl committed:
fix path error

Envl committed:
resolve conflict

Envl committed:
change name to addnewbookpage in index.js

Envl committed:
add props.className to Autocomplete.js

Envl committed:
Style for mobile and desktop

Commit:
connect with firebase

Envl committed:
return number instead string

Envl committed:
upload progress and redirect after upload

Envl committed:
refine animation

Envl committed:
transition time to 2s

Envl committed:
create variable for shadow

Envl committed:
Merge pull request #50 from bt-s/add-book-style-ux

Add book style ux

Envl committed:
create transaction function

Envl committed:
add a clickable button for debug

Commit:
connect to firebase

Envl committed:
resolve conflict

Envl committed:
code clean and make request work

Envl committed:
correct data fields
